sematic/metrics/func_success_rate_metric.py

```diff
@@ -0,0 +1,77 @@
+# Standard Library
+from datetime import datetime
+from typing import Optional, Tuple
+
+# Third-party
+from sqlalchemy.orm import Query, Session, joinedload
+
+# Sematic
+from sematic.abstract_future import FutureState
+from sematic.abstract_system_metric import AbstractSystemMetric
+from sematic.db.models.run import Run
+from sematic.metrics.metric_point import MetricType
+
+FINAL_STATES = {FutureState.RESOLVED, FutureState.FAILED, FutureState.NESTED_FAILED}
```

Comment:
```suggestion
INCLUDED_STATES = {FutureState.RESOLVED, FutureState.FAILED, FutureState.NESTED_FAILED}
```

since there is another final state (`CANCELED`). These are the states that you include in the calculation, so I think that naming is more apt. Not that big of a deal though.